### ISSUE TITLE
Add a sorting function to storage

### ISSUE THREAD

**Issue by ThePiachu** [StrangeLoopGames/EcoSuggestions#1246]
It would be nice for the storage to have a sorting function like Terraria. Basically, anything that could turn this:

![image](https://user-images.githubusercontent.com/964559/95522077-00464080-0980-11eb-94b7-1f8f8c13dfc9.png)

Into this:

![image](https://user-images.githubusercontent.com/964559/95522228-64690480-0980-11eb-8ecd-a7e3d7be83a8.png)

With a single mouse click. Sure, you can accomplish something similar by mass-moving items between two stockpiles, but that's not always ideal. It would make managing the giant Large Lumber Stockpile and anything else that might be coming in the future much easier.

**Comment by lou-vier** [comment]
I totally agree with it, there is big improvements to make in the items sort and items drag&drop. 

It takes many time in this game to organize inventories, and in some other games , there is good shortcuts or buttons that help to organize and transfer items easily.
